cannot install on osx

I get this during `npm install` after installing the other packages required in readme:

```
npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/kain/.nvm/versions/node/v4.2.1/bin/node" "/Users/kain/.nvm/versions/node/v4.2.1/bin/npm" "install"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package reflect-metadata@0.1.3 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer angular2@2.0.0-beta.0 wants reflect-metadata@0.1.2
```

--- a/package.json
+++ b/package.json
@@ -86,7 +86,7 @@
     "highlight.js": "^9.1.0",
     "lodash": "^3.10.1",
     "moment": "^2.10.6",
-    "reflect-metadata": "^0.1.2",
+    "reflect-metadata": "0.1.2",
     "rxjs": "5.0.0-beta.0",
     "systemjs": "^0.19.4",
     "zone.js": "0.5.10"

